source/NVDAObjects/window/_msOfficeChart.py

```diff
@@ -0,0 +1,972 @@
+		return text
+
+	def _get_description(self):
+		count = self.officeChartObject.SeriesCollection().count
+		text=""
+		if count>0:
+			if count == 1:
+				# Translators: Indicates that there is 1 series in a chart.
+				seriesValueString = _( "There is 1 series in this chart" )
+			else:
+				# Translators: Indicates the number of series in a chart where there are multiple series.
+				seriesValueString = _( "There are total %d series in this chart" ) %(count)
+				for i in xrange(1, count+1):
+					# Translators: Specifies the number and name of a series when listing series in a chart.
+					seriesValueString += ", " + _("series {number} {name}").format(number=i, name=self.officeChartObject.SeriesCollection(i).Name)
+				text += seriesValueString
+		else:
+			# Translators: Indicates that there are no series in a chart.
+			text +=_("No Series defined.")
+		return text
+
+	def getChartSegment(self):
+		chartType = self.officeChartObject.ChartType
+		if chartType in (xl3DPie, xl3DPieExploded, xlPie, xlPieExploded, xlPieOfPie):
+			# Translators: A slice in a pie chart.
+			text=_("slice")
+		elif chartType in (xl3DColumn, xl3DColumnClustered, xl3DColumnStacked, xl3DColumnStacked100, xlColumnClustered, xlColumnStacked100, xlColumnStacked):
+			# Translators: A column in a column chart.
+			text=pgettext('chart','column')
+		elif chartType in (xl3DLine, xlLine, xlLineMarkers, xlLineMarkersStacked, xlLineMarkersStacked100, xlLineStacked, xlLineStacked100):
+			# Translators: A data point in a line chart.
+			text=_("data point")
+		else:
+			# Translators: A segment of a chart for charts which don't have a specific name for segments.
+			text=_("item")
+		return text
+
+	def script_activatePosition(self,gesture):
+		# Toggle browse mode pass-through.
+		self.passThrough = True
+		self.ignoreTreeInterceptorPassThrough=False
+		browseMode.reportPassThrough(self)
+	# Translators: Input help mode message for toggle focus and browse mode command in web browsing and other situations.
+	script_activatePosition.__doc__=_("Toggles between browse mode and focus mode. When in focus mode, keys will pass straight through to the application, allowing you to interact directly with a control. When in browse mode, you can navigate the document with the cursor, quick navigation keys, etc.")
+	script_activatePosition.category=inputCore.SCRCAT_BROWSEMODE
+
+	def script_disablePassThrough(self, gesture):
+		rangeStart=self.officeShapeObject.Range.Start
+		self.officeApplicationObject.ActiveDocument.Range(rangeStart, rangeStart).Select()
+		eventHandler.executeEvent("gainFocus", api.getDesktopObject().objectWithFocus())
+
+	__gestures = {
+				"kb:upArrow":"previousElement",
+				"kb:downArrow":"nextElement",
+				"kb:enter": "activatePosition",
+				"kb(desktop):numpadEnter":"activatePosition",
+				"kb:space": "activatePosition",
+				"kb:escape": "disablePassThrough",
+	}
+
+class OfficeChartElement(OfficeChartElementList):
```

Comment:
I find the name of this particular class confusing. I'm assuming that this element represents all the stuff that is not a series? perhaps call it OfficeChartElementMeta or something? saying OfficeChartElement makes it look too generic.